front/app/containers/MainHeader/Components/NotificationMenu/components/IdeaAssignedToYouNotification/index.tsx

```diff
@@ -60,34 +59,19 @@ const IdeaAssignedToYouNotification = ({ notification }: Props) => {
     }
   };
 
-  const getLinkTo = () => {
-    if (!isNilOrError(authUser)) {
-      if (isAdmin(authUser)) {
-        return '/admin/ideas';
-      } else if (projectId && isProjectModerator(authUser, projectId)) {
-        return `/admin/projects/${projectId}/ideas`;
```

Comment:
A link to the general input manager is pretty useless imo and for moderators the link was broken. So since we can directly link to ideas in the admin, I'm doing this instead.

Reply:
While checking this, I discovered [two other broken links](https://github.com/CitizenLabDotCo/citizenlab/compare/TAN-1318/broken-links-input-manager?expand=1) to the input manager. Fixing in a different PR.

Reply:
Will add typing the routes as a task for platform week, hopefully we get to it.